client/README.md

```diff
@@ -192,3 +192,746 @@ For example:
+
+## URL Route
+
+"/community/join-groups"
+
+## Current Status
+
+The page is static and does not pull any data from the backend.
+
+The page needs to make API calls to get group locations (city and state) so that the groups rendered align with the location specified by the user.
+
+This page needs data from the Groups model on the back-end.
+
+## Future Development
+
+1. Make the search bar, "sort by" filter, and "See More" features function.
+2. Pills should render group data pulled from the back-end. When clicked, pills should change color and redirect the user to that group's page.
+
+# Community Groups Page
+
+## Description
+
+After a user clicks on a specific group, they are redirected to this page where they can learn more about a specific group, see different user comments and hashtags, a description of the page, members that are part of the group, and upcoming events.
+
+## URL Route
+
+"/community/view-group"
+
+## Current Status
+
+The page has all necessary components but only renders mock data. The page is mostly mobile responsive. The page needs to fetch group data from the backend to search the group in the group modals.
+
+## Future Development
+
+Different users should be able to like a post and comment on the post, click on the hashtag, click "join" to join the group, click on members to see different members, and click on upcoming events.
+
+# Community People Page
+
+## Description
+
+This page allows the users to view members by location. The user can also search for groups, and filter search results with the sort feature.
+
+## URL Route
+
+"/community/view-group"
+
+## Current Status
+
+The page is currently static. The search bar and 'sort by' features do not work. The user should be able to click "See More." "Discover Groups" and "Connect With Other Moms" should each link to the appropriate pages.
+
+## Future Development
+
+1. Build the 'sort by' feature
+2. Build the search bar
+3. Add event listener so that pills change color when clicked
+4. Header buttons should by dynamic
+5. "See All" should work
+
+# Forum Page
+
+## Description
+
+The forum page allows the user to search for topics and hashtags, and sort by different criteria. The user can also add favorite topics and hashtags that they "heart" (like) as well.
+
+## URL Route
+
+"/forum-page"
+
+## Current Status
+
+The forum page is currently all static but the tabs on the top will link to the correct page when clicked on. The search bar, "My favorite topics" and "Discover Topics" are static.
+
+## Future Development
+
+1. Build the search bar so that users can search
+2. "My favorite topics" and "Discover topic" should link properly
+3. "Pills" should change color when clicked
+
+# Forum - Hashtag
+
+## Description
+
+This page returns results for the hashtag searched by the user.
+
+## URL Route
+
+"/forum-page-hashtag"
```

Comment:
Please put this in a backtick quote, so it looks like this:
`/forum-page-hashtag`